aea/decision_maker/gop.py

```diff
@@ -0,0 +1,811 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
```

Comment:
At the moment this is just used for TAC, so makes no sense to use it as default.